product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -103,6 +103,7 @@ Similar to the `consensus_grace_period`, a `read_consensus_grace_period` option
 
 ### Multi-host connection strings
 
-The PostgreSQL C client library (libpq) allows you to specify multiple host names in a single connection string for simple failover. This is also supported by client libraries (drivers) in some other programming languages. It works well for failing over across PGD Proxy instances that are down or inaccessible. 
+The PostgreSQL C client library (libpq) allows you to specify multiple host names in a single connection string for simple failover. This ability is also supported by client libraries (drivers) in some other programming languages. It works well for failing over across PGD Proxy instances that are down or inaccessible. 
 
 However, if the PGD Proxy instance is accessible but doesn't have access to the write leader, or the write leader for a given instance doesn't exist (that is, because there's no write leader for the given PGD group), the connection simply fails. No other hosts in the multi-host connection string is tried. This behavior is consistent with the behavior of PostgreSQL client libraries with other proxies like HAProxy or pgbouncer.
```

Comment:
This is a very long sentence. Here's one suggestion for how to make it easier to read:

However, if either of these conditions is true, the connection simply fails:

- The PGD Proxy instance is accessible but doesn't have access to the write leader. 
- The write leader for a given instance doesn't exist (that is, because there's no write leader for the given PGD group). 
When the connection fails, no other hosts in the multi-host connection string is tried. This behavior is consistent with the behavior of PostgreSQL client libraries with other proxies like HAProxy or pgbouncer.